### PR TITLE
Added note listing page UI

### PR DESCRIPTION
**Pull request by soorajjtk**

### CODE REVIEW

File: app/javascript/src/components/Common/Menubar.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    <MenuBar showMenu title={title}>
+      {options?.heads?.map(head => (
+        <MenuBar.Block
+          label={head.title}
+          key={head.title}
+          count={head.count}
+          onClick={() => setCategory(head.title)}
+        />
+      ))}
+
```
Comment (yedhink) [inline app/javascript/src/components/Common/Menubar.jsx:21]: Please don't add unnecessary blank space in JSX! It's visually not helpful to add a blank line in JSX. Rather it's more confusing given that it's the presentation layer.

PS: It's a good practice to add blank in lines in JS to provide visual hierarchy. Here I am specifically talking about JSX.

File: app/javascript/src/components/Common/Menubar.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
```
Comment (yedhink) [inline app/javascript/src/components/Common/Menubar.jsx:5]: Why you have you imported from `neetoui` at some places and `@bigbinary/neetoui` at some other places? Stick onto one depending on the webpack aliases you've set.

File: app/javascript/src/components/Dashboard/Notes/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
+import { Typography, Dropdown, Tooltip, Avatar } from "@bigbinary/neetoui";
+import PropTypes from "prop-types";
+
+const Card = props => {
+  const { note, action } = props;
+  return (
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Card.jsx:9]: Always add a newline before the `return` statement. It denotes existing out of a scope.

File: app/javascript/src/components/Dashboard/Notes/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        </button>
+        <div className="flex items-center space-x-2">
+          <Clock color="#68737D" size={14} />
+          <Tooltip position="bottom" content={note?.time}>
+            <Typography style="body3" className="text-gray-500">
+              Created {note?.createdAgo} hours ago
+            </Typography>
+          </Tooltip>
+          <Avatar
+            onClick={function noRefCheck() {}}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Card.jsx:40]: Use arrow function, always, unless there's a requirement to access `this`.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 5]
```diff
@@ -1,16 +1,34 @@
...
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
 import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Container, Header } from "neetoui/layouts";
 
 import notesApi from "apis/notes";
 import EmptyState from "components/Common/EmptyState";
+import Menubar from "components/Common/Menubar";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:9]: Add an index.js to covert this into a single import statement.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 6]
```diff
@@ -1,16 +1,34 @@
...
+  });
+  return `${weekday}, ${time}`;
+};
+
+const calculateHoursAgo = date => {
+  const dateTime = new Date(date);
+  const currentDateTime = new Date();
+  const hours = Math.abs(currentDateTime.getTime() - dateTime.getTime()) / 36e5;
+  return Math.round(hours);
+};
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:31]: In BigBinary we make use of https://day.js.org/. You can use that over here. Also these utils functions ought to be moved into an `utils.js` within the closest scope that's `Notes`.

**Review by yedhink — COMMENTED** [review]
@soorajjtk _a Have added few comments. PS: The title of the PR is wrong! Read: https://handbook.neetokb.com/articles/git-and-github-essentials#creating-pull-request

- I haven't pinpointed same review request at different code blocks within this PR. You can go through the PR and figure out where all similar changes are required.
- You can make the changes via a separate PR or as part of a PR that you are already working on. You can make the call on that.
- Lemme know if you need any clarifications.

Would highly recommend going through the best practices section(skip `ramda` part) in https://academy.bigbinary.com/learn-react if you haven't done so already.

File: app/javascript/src/components/Dashboard/Notes/index.jsx (hunk 6 above)
Comment (soorajjtk) [reply app/javascript/src/components/Dashboard/Notes/index.jsx:31]: Resolved all the above mentioned comments in PR #13

**Pull request merged**